.github/workflows/ci.yml

```
@@ -6,26 +6,36 @@ on:
-          bun-version: latest
+          graphite_token: ${{ secrets.GRAPHITE_CI_TOKEN }}
 
-      - name: Install dependencies
-        run: bun install --frozen-lockfile
+  lint:
+    name: Lint & Type Check
+    runs-on: ubuntu-latest
+    needs: optimize_ci
+    if: needs.optimize_ci.result != 'failure' && needs.optimize_ci.outputs.skip == 'false'
 
       - name: Run Biome
```

actionlint (1.7.8)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)